lang/python/django-compressor/Makefile

```diff
@@ -27,7 +27,7 @@ define Package/django-compressor
   CATEGORY:=Languages
   TITLE:=Compresses linked and inline JavaScript or CSS into single cached files.
   URL:=http://django-compressor.readthedocs.org/
-  DEPENDS:=+python +django
+  DEPENDS:=+python +django +python-rcssmin
```

Comment:
oh
so this is where this was needed ?

curios; is there a new `django-compressor` that doesn't require this ?

Reply:
2.2 is the latest release, from 2017.  Judging from the latest commits, it is currently maintained.  Its dependencies, taken from `setup.py` are listed as: `'django-appconf >= 1.0', 'rcssmin == 1.0.6', 'rjsmin == 1.0.12'`.  There is no package for rjsmin, but as long as you don't call the jsmin filter for compressor, you won't need it.  I've run this to find out what was actually being called:
```
$ egrep -r compressor.filter
usr/share/seafile/seafile-server/seahub/seahub/settings.py:    'compressor.filters.css_default.CssAbsoluteFilter',
usr/share/seafile/seafile-server/seahub/seahub/settings.py:    'compressor.filters.cssmin.CSSMinFilter',
```

Now that you got me into writing this, I will add the `django-appconf` dependency, and a note in the package help about the need to build rjsmin to use JavaScript (we don't have a rjsmin package currently, and I don't think we should add a python 2 version of it anyway).

Reply:
This seems to be getting more complicated.
It also looks like the initial work was done in a bit of haste.

Oh well, let's see.

I will try to allocate more time on this, in the next few weeks.